### ISSUE TITLE
Механика газов

### ISSUE THREAD

**Issue by Eliotopg** [Anuken/Mindustry-Suggestions#3070]
### Describe the content or mechanics you are proposing.

Газы:

1)Природный
2)Ядовитый
3)Проводящий
4)Инертный
5)Взрывчатый
6)Парализующий
7)Светящийся

Блоки:

1) Газовый бур
2)Сильный газовый бур
3)Газовый фонарь
4)Огнемёт
5)Газомет(отравляющий)

Механики:

1)Взрыв газа
2)Отравление газом
3)Исцеление газом
4) Паралич газом

Руды:

1)Твердая газовая руда
2)Газовая скважина

Заранее спасибо разработчикам.

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

Газы сделают своё дело закончив ветку ресурсов.

Новый материал
Новые функции
Заполнение ниши новых руд
Заполнение потребности в новых турелях
Экстрим
Отсутствие газа в clasic.
Ещё раз спасибо разработчикам заранее.

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by RealYeet** [comment]
Translated 

Describe the content or mechanics you are proposing.

 Gases:

 1) Natural
 2) Poisonous
 3) Conductive
 4) Inert
 5) Explosive
 6) Paralyzing
 7) Luminous

 Blocks:

 Gas drill
 2) Strong gas drill
 3) Gas lantern
 4) Flamethrower
 5) Gas cannon (poisonous)
 Mechanics:

 1) Gas explosion
 2) Gas poisoning
 3) Gas Healing
 4) Gas paralysis

 Ores:

 1) Solid gas ore
 2) Gas well

 Thanks in advance to the developers.

 Describe how you think this content will improve the game.  If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

 Gases will do their job by finishing the resource branch.

 New material
 New features
 Filling the niche of new ores
 Filling the need for new turrets
 Extreme
 Lack of gas in the clasic.
 Thanks again to the developers in advance.

**Comment by Eliotopg** [comment]
Thanks for the correct translation.


вт, 5 окт. 2021 г., 10:38 PM RealYeet ***@***.***>:

**Comment by MrDuck557** [comment]
in other words:
added complexity
for things that can already be achieved through mods of existing content

**Comment by Kyllingene** [comment]
> Gases wouldn't be a bad feature tho, we already have solids and liquids

Although it might appear otherwise, these are two completely unrelated, independent sentences, joined unfortunately with a punctuation mark. They may have been intended to be congruous, but they are not. The second sentence simply states that we have solids and liquids, and makes no attempt to explain why that has any bearing on the subject at hand.

The simple fact that we have two states of matter does not require that we include a third; if it did, we would require plasma as well, and what about semisolids, and all other manner of in-betweens that we could shoehorn into the game to make it as complex and unwieldy as possible.

**Comment by Feilong-Mindustry** [comment]
I think it's a stretch to ask Anuken for this level of added features. Could be cool if implemented, but there are so many things most players would like to see before this one.

Paralyzing or poisoning mechanical units with gas doesn't sound right. Did you take your time to think through mechanics? There is a vast field for imagination. For example implementing gases in resource and unit production, new reactors and unique logistic features.

P.S Kyllingene, you are a dork.

**Comment by Kyllingene** [comment]
I agree; I've been reading Benjamin Franklin's autobiography, and it seems to have infected my speech.

**Comment by oinkymoo** [comment]
"new ores" and "new turrets" are not niches that need to be filled. what purpose would they serve
